sdcm/mgmt.py

```diff
@@ -144,9 +144,6 @@ def start(self, continue_task=True):
         if not continue_task:
             cmd += " --no-continue"
         self.sctool.run(cmd=cmd, is_verify_errorless_result=True)
-        list_all_task_status = [s for s in TaskStatus.__dict__ if not s.startswith("__")]
-        list_expected_task_status = [status for status in list_all_task_status if status != TaskStatus.STOPPED]
-        return self.wait_for_status(list_status=list_expected_task_status, timeout=30, step=3)
```

Comment:
are you sure about remove this `wait_for_status` won't it break some of the tests ?

Reply:
@ShlomiBalalis I'm still waiting for answer on this one...